util/cosmos/misc.js

```diff
@@ -17,7 +17,7 @@ export function assertOk (res) {
     throw new Error(message)
   }
```

Comment:
Not quite sure if the error format of the new endpoint still follows the above flow.

Reply:
https://mainnet-node.like.co/cosmos/tx/v1beta1/txs/0022328B1B61161943373D4C76F386BF8F036C947A30D588243804333F37CF12
just found a random failed tx in explorer, need to modify a bit i guess